src/Bridge/Symfony/XliffConverter.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony;
+
+use Symfony\Component\Translation\MessageCatalogue;
+use Translation\SymfonyStorage\Dumper\XliffDumper;
+use Translation\SymfonyStorage\Loader\XliffLoader;
+
+/**
+ * Utility class to convert between a MessageCatalogue and XLIFF file content.
+ */
+final class XliffConverter
```

Comment:
@prolic that's what https://github.com/php-translation/symfony-storage/blob/master/src/XliffConverter.php does (it's exactely the same)